package.json

```diff
@@ -10,15 +10,15 @@
     "uuid": "~8.3.2"
   },
   "devDependencies": {
-    "@supercharge/eslint-config-typescript": "~1.0.1",
-    "@supercharge/tsconfig": "~2.0.0",
+    "@supercharge/eslint-config-typescript": "~2.3.0",
+    "@supercharge/tsconfig": "~4.0.0",
```

Comment:
bumping `@supercharge/tsconfig` from `2.0.0` to `4.0.0` is a breaking change. Version `4.0.0` requires Node.js 16 or later.

Can you please revert the dependency bumps in your branch?

Reply:
Apologies, reverted now.

Reply:
Thanks 👍